ush/config_defaults.yaml

```diff
@@ -2901,8 +2901,10 @@ cpl_aqm_parm:
   # DO_AQM_GEFS_LBCS:
   # Add GEFS aerosol LBCs to chemical LBCs
   #
-  # DO_AQM_SAVE_AIRNOW_HIST:
-  # Save bias-correction airnow training data
```

Comment:
This is not the same as AQM_AIRNOW_HIST_DIR. This is a flag to determine whether or not the training data is saved. I think we don't have to remove this.